### PR TITLE
Bump Jinja version to 2.10.1

### PR DESCRIPTION
**Pull request by yvaucher**
Bump Jinja version

and fix issue with libc-ares2

### CODE REVIEW

**Comment by simahawk** [comment]
@yvaucher :red_circle:

**Comment by yvaucher** [comment]
@simahawk Yep, on jessie seems the package libc-ares2 is not registered with the proper key.

I tried `apt key-update` but it didn't solve it.
Using `--force-yes` allow us to fix the build, but that seems to be a temporary fix.

File: install/package_odoo_9.0_10.0.sh [hunk 1]
```diff
@@ -2,6 +2,8 @@
 set -eo pipefail
 
 apt-get update
+# authentication error for libc-ares2
```
Comment (simahawk) [inline install/package_odoo_9.0_10.0.sh]: can you open an issue to not forget about this?

Comment (yvaucher) [reply install/package_odoo_9.0_10.0.sh]: https://github.com/camptocamp/docker-odoo-project/issues/108

**Pull request merged**